config-connector/solutions/iam/kpt/service-account/iampolicy.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMPolicy
+metadata:
+  labels:
```

Comment:
I think we don't want to set a random label here.

Reply:
Discussed offline. Removed.